google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -224,27 +226,29 @@ def copy_file source_bucket_name, source_file_path,
         def rewrite_file source_bucket_name, source_file_path,
```

Comment:
The only difference is the presence of the `rewriteToken` argument. There was a lot of churn to get this functionality, and I didn't want to add even more to combine all the calls to these two methods. We could do this as part of a refactor, but it doesn't need to be in this PR either. (I have another PR queued up once this is merged.)